app/views/documents/fields/_govspeak_input.html.erb

```diff
@@ -20,8 +20,7 @@
       contextual_guidance: "document-contents-guidance",
       govspeak_path: govspeak_preview_path(edition.document),
       edition: edition,
-      error_items: issues&.items_for(field.id.to_sym),
-      paste_html_to_govspeak: current_user.has_permission?(User::PRE_RELEASE_FEATURES_PERMISSION)
```

Comment:
I think we should just change this to true for now rather than remove the conditional aspects. In the early days of this it would give us a bit more flexibility to turn it off and on and would leave both iterations in the component-guide.

Reply:
I've changed this to true now.